### PR TITLE
feat(shared): [SPB-1999] export used helpers

### PR DESCRIPTION
**Pull request by PeterMK85**
[SPB-1999]

### Description

Collected all use cases from `billing-frontend` and `staff-portal` and expose used helpers

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- ~[ ]~ n/a Annotate all `props` in component with documentation
- ~[ ]~ n/a Create `examples` for component
- ~[ ]~ n/a Ensure that deployed demo has expected results and good examples
- ~[ ]~ n/a Ensure that tests pass by running `yarn test`
- ~[ ]~ n/a Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- ~[ ]~ n/a Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

[SPB-1999]: https://toptal-core.atlassian.net/browse/SPB-1999

### CODE REVIEW

**Review by OleksandrNechai — APPROVED** [review]

File: packages/picasso/src/index.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export { default, BaseProps } from '@toptal/picasso-shared'
+export { default, BaseProps, ColorType, SpacingType, alpha, SizeType } from '@toptal/picasso-shared'
```
Comment (denieler) [inline packages/picasso/src/index.ts:1]: for types we need to use `export type { Color Type }` ☝️ 

for `alpha` utility tool, I would probably export it as utility object or something like that

Comment (PeterMK85) [reply packages/picasso/src/index.ts]: @denieler 

Adjusted the type export, about alpha not sure current FX plans. Do you have a ticket on your roadmap. I would export the color utility together probably under utils

File: packages/picasso/src/Grid/index.ts [hunk 2]
```diff
@@ -4,3 +4,4 @@ import { Props } from './Grid'
 
 export { default } from './Grid'
 export type GridProps = OmitInternalProps<Props>
+export { GridSize } from '@material-ui/core/Grid'
```
Comment (denieler) [inline packages/picasso/src/Grid/index.ts:7]:
```suggestion
export type { GridSize } from '@material-ui/core/Grid'
```
but probably you need to add this file to `.prettierignore`

Comment (PeterMK85) [reply packages/picasso/src/Grid/index.ts:7]: added :+1:

File: .prettierignore [hunk 3]
```diff
@@ -1,21 +1,22 @@
 # Those files need to be removed from here after this
 # version of prettier https://github.com/prettier/prettier/issues/7263 will
 # be supported in prettier-standard
-packages/picasso/src/index.ts
```
Comment (PeterMK85) [inline .prettierignore:4]: sorted and added `packages/picasso/src/Grid/index.tsx`

File: .prettierignore [hunk 4]
```diff
@@ -1,21 +1,22 @@
 # Those files need to be removed from here after this
 # version of prettier https://github.com/prettier/prettier/issues/7263 will
 # be supported in prettier-standard
```
Comment (PeterMK85) [inline .prettierignore:3]: Shared with @denieler that its no longer required.

Example setup:
https://github.com/toptal/staff-portal/blob/master/package.json#L19

CC:
@havenchyk

File: packages/picasso/src/utils/index.ts [hunk 5]
```diff
@@ -1,5 +1,15 @@
+import { alpha, lighten, darken } from '@toptal/picasso-shared'
+
 import * as TransitionUtils from './Transitions'
 
+const colorTools = {
```
Comment (havenchyk) [inline packages/picasso/src/utils/index.ts:5]: we didn't group other utils under namespace. Do you think it's valuable @PeterMK85 ?

Comment (PeterMK85) [reply packages/picasso/src/utils/index.ts:5]: Decided by @denieler :) first Ive exported as is :)

**Review by denieler — APPROVED** [review]

File: packages/picasso/src/utils/index.ts (hunk 5 above)
Comment (denieler) [reply packages/picasso/src/utils/index.ts:5]: @havenchyk  idk, I proposed this, so user maybe can easier navigate in the exported things, but it's up to discussion 😃

Comment (havenchyk) [reply packages/picasso/src/utils/index.ts:5]: I'm afraid only about the possible breaking change if we decide to change it in the future. 

For consistency I would do it as `export { alpha, lighten, darken } from '@toptal/picasso-shared'`

@denieler do you think it's easier to have a namespace?

Comment (havenchyk) [reply packages/picasso/src/utils/index.ts:5]: @PeterMK85 let's rename it to `colorUtils` and we're good to go

**Review by havenchyk — APPROVED** [review]

**Pull request merged**